monolith/views/users.py

```diff
@@ -11,17 +11,20 @@ def _users():
     return render_template("users.html", users=users)
 
 
-@users.route('/create_user', methods=['GET', 'POST'])
+@users.route('/signup', methods=['GET', 'POST'])
 def create_user():
+    if current_user is not None and hasattr(current_user, 'id'):
```

Comment:
I was reading about flask login and I found this https://flask-login.readthedocs.io/en/latest/#anonymous-users.
Should we be checking if current_user is not None or if current_user.is_anonymous is True?

Reply:
Works! I'll update the branch